math/mathcore/src/TMath.cxx

```diff
@@ -3151,19 +3154,788 @@ Double_t TMath::VavilovDenEval(Double_t rlam, Double_t *AC, Double_t *HC, Int_t
    return v;
 }
 
-
 //explicitly instantiate template functions from VecCore
-#ifdef R__HAS_VECCORE
-#include <Math/Types.h>
+//#ifdef R__HAS_VECCORE
```

Comment:
Removing these guards also break compilation and go against the advice that we gave that your code should work both with VecCore enabled and disabled.

Reply:
Yes, I agree. 
This was done just to test on my machine making sure that the lines after these are compiled.
Will explain on our mattermost channel why I did so.